docs/content/tendermint-and-cosmos.rst

```diff
@@ -0,0 +1,93 @@
+Tendermint & Cosmos
+===================
+
+We're building a lot of software under two different GitHub organizations:
+
+- `Tendermint <https://github.com/tendermint>`__ and,
+- `Cosmos <https://github.com/cosmos>`__
+
+Repositories are classified as either "Core" or "Secondary" where the former generally produce binaries or code consumable by users and the latter provide building blocks for the former. Note that the status of "Secondary" repos is subject to change (i.e., they into "Core" repos) as we consolidate over time.
+
+- TODO, still missing a bunch of relevant public repos
+
+Tendermint
+----------
+
+Core Repositories
+~~~~~~~~~~~~~~~~~
+
+blurb 
+
+Tendermint
+^^^^^^^^^^
+
+Tendermint Core is Byzantine Fault Tolerant (BFT) middleware that takes a state transition machine - written in any programming language - and securely replicates it on many machines.
```

Comment:
I guess this will be extended to a more detailed explanation?

Reply:
This applies to all other sub-sections

Reply:
Not really, because those extended explanations can be in the introduction of the Read The Docs. I'm not too sure what to do here, in the sense that there will inevitably be some overlap